app/commands/auth/accounts/send_authentication_email.rb

```
@@ -5,22 +5,24 @@ module Accounts
     class SendAuthenticationEmail < ApplicationCommand
       prepend SimpleCommand
 
-      attr_reader :email
+      attr_reader :email, :id
 
-      def initialize(email:)
+      def initialize(email:, id:)
         @email = email
+        @id = id
       end
 
       def call
         with_exception_handle do
-          @account = Account.create_user_for_provider(email, 'magic_link')
-
+          @account = if email.present?
+                       Account.create_user_for_provider(email, 'magic_link')
+                     else
+                       Account.find(id)
+                     end
```

I don't know if rails already does that, but i think we should raise an error in case account is nil

---

and it would be good to test this too